src/application/wizard/service.ts

```diff
@@ -75,6 +76,7 @@ export class WizardService {
         submissionId: SubmissionId,
         file: FileUpload,
         fileSize: number,
+        pubsub: PubSub,
     ): Promise<Submission> {
         const submission = await this.submissionService.get(submissionId);
```

Comment:
Should probably publish a 1or 0 progress event here to alert client the process has begun

Reply:
Not sure. I think it makes sense to probably open a socket on the first upload and filter based on filename. You will get events from 1..last updates